crypto/evp/p_lib.c

```diff
@@ -237,6 +237,17 @@ void *EVP_PKEY_get0(const EVP_PKEY *pkey)
     return pkey->pkey.ptr;
 }
 
+const unsigned char *EVP_PKEY_get_hmac(const EVP_PKEY *pkey, int *len)
```

Comment:
Does this stand a *chance* of making it into 1.1?  If not, we should be looking at other things ☺